src/tools/r2rdump/R2RSection.cs

```diff
@@ -42,8 +42,13 @@ public enum SectionType
         /// </summary>
         public int Size { get; set; }
 
-        public R2RSection(SectionType type, int rva, int size)
+        private bool _ignoreSensitive;
+        public bool ShouldSerializeRelativeVirtualAddress() { return !_ignoreSensitive; }
```

Comment:
Nit: You can write single-expression methods and properties as
`public bool ShouldSerializeRelativeVirtualAddress() => !_ignoreSensitive;`